src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl

```diff
@@ -86,15 +87,13 @@ function ImmersedBoundaryGrid(grid, ib::AbstractGridFittedBottom{<:OffsetArray})
 end
 
 function validate_ib_size(grid, ib)
-    Nx, Ny, _ = size(grid)
-    Hx, Hy, _ = halo_size(grid)
+    bottom_height_size = total_size((Center, Center, Nothing), grid)[1:2]
 
-    bottom_height_size = (Nx, Ny) .+ 2 .* (Hx, Hy)
-
-    # Check that the size of a bottom field are 
-    # consistent with the size of the field
+    # Why do we use `any` here? Is this a bug?
     any(size(ib.bottom_height) .!= bottom_height_size) && 
```

Comment:
I guess this is interchangeable with `!(size(ib.bottom_height) == bottom_height_size)`

Reply:
Oh. `!=` will do:

```julia
julia> (1, 2, 3) != (1, 2, 4)
true

julia> (1, 2, 3) != (1, 2, 3)
false
```